### PR TITLE
Add consistent titles, favicon and nest logo to the apps

### LINKED ISSUE #55: More consistent title for demo teal apps
### Feature description

Linked to https://github.com/insightsengineering/teal.gallery/issues/13. For more consistent look/feel of demo teal apps, suggest to make the following changes

For all apps, show text "Teal @ Roche" to the top-right, and if possible add the teal hex.
@telepath37 or do you think it should show "NEST @ Roche" and the nest hex instead?

For text to show at top-left
1. basic-teal - My first teal app
2. exploratory - Example teal app for general dataset exploration with teal.modules.general 
3. safety - Example teal app focusing on safety analysis of clinical trial data with teal.modules.clinical
4. efficacy - Example teal app focusing on efficacy analysis of clinical trial data with teal.modules.clinical
5. patient-profile - Example teal app focusing on patient-level analysis of clinical trial data with teal.modules.clinical
6. early-dev - Example teal app focusing on analysis of early-phase clinical trial data with teal.osprey
7. longitudinal - Example teal app focusing on analysis of longitudinal clinical trial data with teal.goshawk
8. RNA-seq - Example teal app focusing on analysis of RNA-seq data with teal.modules.hermes
9. python - Example teal app using python dataset connector

As we have quite a few presentations/workshops upcoming, please consider prioritizing this issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by vedhav**
Closes #55

### CODE REVIEW

**Comment by vedhav** [comment]
@lcd2yyz @pawelru I heard that we have a new NEST logo coming soon, If we already have it please let me know so we can replace the old NEST logo over here as well.

**Comment by vedhav** [comment]
@lcd2yyz Since the titles and `NEST @ Roche` can sometimes be too long they will be wrapped when there is not enough space.
Kapture 2023-08-29 at 12 35 57

**Comment by pawelru** [comment]
> @lcd2yyz @pawelru I heard that we have a new NEST logo coming soon, If we already have it please let me know so we can replace the old NEST logo over here as well.

We will be having tomorrow a catch-up with the designer. I don't expect to have it ready to use. It's still more in the discovery phase. If that's the only thing that blocks this PR then please don't wait and we will update the logos afterwards.
I also saw that you are considering re-record gifs. For this I would suggest to wait. Changing logos will make relatively big difference.

**Comment by lcd2yyz** [comment]
Thanks for checking @vedhav. Agree with @pawelru, new logo won't be ready to use this time around.

The wraparound (upon window width reduction) looks fine to me. However, the NEST logo (shown on the landing page) looks a bit too big to me in the video. Can we try reduce the size by ~25% to leave some more white space margin from the top/bottom.

File: RNA-seq/app.R [hunk 1]
```diff
@@ -64,12 +64,13 @@ app <- init(
...
-    style = "margin-bottom: 2px;",
-    tags$h1(
-      "Example App with teal.modules.hermes modules",
-      tags$span("SPA", class = "pull-right")
+  header = tags$span(
+    style = "display: flex; align-items: center; justify-content: space-between;",
+    tags$h2("Example teal app focusing on analysis of RNA-seq data with teal.modules.hermes"),
+    tags$span(
+      style = "display: flex; align-items: center;",
+      tags$img(src = "nest.png", alt = "NEST logo", height = "60px"),
```
Comment (lcd2yyz) [inline RNA-seq/app.R:72]: Reduce 60px to maybe 45px for all?

**Review by lcd2yyz — APPROVED** [review]
The rest looks good

**Comment by lcd2yyz** [comment]
> I also saw that you are considering re-record gifs. For this I would suggest to wait. Changing logos will make relatively big difference.

Hi @pawelru, I actually think we should proceed with re-recording gifs now with the newly changed contents. The changes, especially in titles line and landing page, are quite visible from the demo app gallery panels. So in terms of publicizing efforts, the sooner we get to consistency the better. Sure, the logo might change in maybe 2 months, but it won't really affect the app content or the understanding of the app itself in anyway.
If the eventual logo change really bothers you, and that we want to avoiding the need to re-record yet again, I'm fine with removing the NEST logo altogether (and just leave is as pure text of "NEST @ Roche"), so it won't show at all in the recording. We'd just need to update the favicon when the new logo design is finalized.

**Pull request merged**